Source/SwiftLintFramework/Rules/Lint/PrivateSubjectRuleExamples.swift

```diff
@@ -100,7 +100,14 @@ internal struct PrivateSubjectRuleExamples {
                     CurrentValueSubject<Bool, Never>(true)
             }
             """#
-        )
+        ),
+        Example(
+            """
+            func foo() {
+                let goodSubject = PassthroughSubject<Bool, Never>(true)
+            }
+            """
+        ),
```

Comment:
This evil trailing comma is causing the test failures. 😬

Reply:
muscle memory, on my daily job we require it 😅